common/src/main/resources/META-INF/native-image/io.netty/common/reflect-config.json

```diff
@@ -0,0 +1,266 @@
+[
+  {
+    "name": "io.netty.util.AbstractReferenceCounted",
+    "fields": [
+      {
+        "name": "refCnt",
+        "allowUnsafeAccess": true
+      }
+    ]
+  },
+  {
+    "name": "io.netty.util.DefaultAttributeMap"
+  },
+  {
+    "name": "io.netty.util.DefaultAttributeMap$DefaultAttribute"
+  },
+  {
+    "name": "io.netty.util.HashedWheelTimer"
+  },
+  {
+    "name": "io.netty.util.HashedWheelTimer$HashedWheelTimeout"
+  },
+  {
+    "name": "io.netty.util.ReferenceCountUtil",
+    "allDeclaredMethods": true
```

Comment:
Here the only method that we need should be `touch`

Reply:
Fixed.